pkgs/games/heroic/fhsenv.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, buildFHSUserEnv
+, heroic-unwrapped
+, extraPkgs ? pkgs: [ ]
+, extraLibraries ? pkgs: [ ]
+}:
+
+buildFHSUserEnv {
+  name = "heroic";
+
+  runScript = "heroic";
+
+  targetPkgs = pkgs: with pkgs; [
+    heroic-unwrapped
+
```

Comment:
Can you please remove this extra line? With that, I can throw the approve for this PR.